### ISSUE TITLE
tools/pg_diff truncates the immediately-preceding context line

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#678]
When adding a `password` column to `assessment_access_rules`, `pg_diff` returns:

```text
...
      mode: enum_mode
      n
-     password: text
      role: enum_role
...
```

The single `n` on the line by itself should be `number: integer`.

**Comment by nwalters512** [comment]
weird... will investigate

**Comment by mwest1066** [comment]
Fixed by #679
